test/info-extras-test.js

```diff
@@ -225,3 +225,51 @@ describe('extras.getRelatedVideos()', () => {
     });
   });
 });
+
+describe('extras.getLikes()', () => {
+  it('Returnes like count', (done) => {
+    fs.readFile(path.resolve(__dirname,
+      'files/videos/_HSylqgVYQI-regular/watch.html'), 'utf8', (err, html) => {
+      assert.ifError(err);
+      const likes = extras.getLikes(html);
+      assert.ok(likes);
+      done();
+    });
+  });
+
+  describe('With no likes', () => {
+    it('Does not return likes', (done) => {
+      fs.readFile(path.resolve(__dirname,
+        'files/videos/KKzOh0MRuZE-no-likes-or-dislikes/watch.html'), 'utf8', (err, html) => {
+        assert.ifError(err);
+        const likes = extras.getLikes(html);
+        assert.ok(likes === null);
```

Comment:
use `assert.equal`. it's best used when comparing equality, and has better error reporting if the arguments turn out to be unequal.